vllm_spyre/platform.py

```diff
@@ -77,26 +78,28 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         # Override --max-num-seqs to the biggest warmup batch size
         # And override --max-model-len to the biggest warmup sequence
         cls._warmup_shapes = None
-        spyre_warmup_shapes = cls.get_warmup_shapes(scheduler_config)
+        spyre_warmup_shapes = \
+            cls.get_warmup_shapes(scheduler_config)
         max_batch_size = 0
         max_seq_len = 0
         for shape in spyre_warmup_shapes:
             max_batch_size = max(max_batch_size, shape['batch_size'])
             max_seq_len = max(max_seq_len,
                               shape['prompt_length'] + shape['new_tokens'])
 
-        if envs.VLLM_USE_V1:
-            if envs_spyre.VLLM_SPYRE_USE_CB:
-                # For continuous batching we use max_num_seqs to control
-                # the max batch size respecting AIU Spyre KV cache size
-                scheduler_config.max_num_seqs =\
-                    envs_spyre.VLLM_SPYRE_MAX_BATCH_SIZE
-                # ToDo: this function check_and_update_config is called twice:
-                # 1st time scheduler_config.max_num_seqs is what user sets
-                # 2nd time scheduler_config.max_num_seqs is 128
-            else:
-                # The v0 scheduler will run out of blocks if this is overridden
-                scheduler_config.max_num_seqs = max_batch_size
+        # if envs.VLLM_USE_V1:
+        #     if envs_spyre.VLLM_SPYRE_USE_CB:
+        #         # For continuous batching we use max_num_seqs to control
+        #         # the max batch size respecting AIU Spyre KV cache size
+        #         scheduler_config.max_num_seqs =\
+        #             envs_spyre.VLLM_SPYRE_MAX_BATCH_SIZE
+        #         # ToDo: this function check_and_update_config is called twice:
+        #         # 1st time scheduler_config.max_num_seqs is what user sets
+        #         # 2nd time scheduler_config.max_num_seqs is 128
+        #     else:
+        #         # The v0 scheduler will run out of
+        #           blocks if this is overridden
+        #         scheduler_config.max_num_seqs = max_batch_size
```

Comment:
not 100% sure we need this?
```
        #         scheduler_config.max_num_seqs = max_batch_size
```

Reply:
For the static batching use case the max batch size is still set with `VLLM_SPYRE_WARMUP_BATCH_SIZES`, so we override it to make sure the scheduling config is consistent. (e.g. setting both `--max-num-seqs=1` and `VLLM_SPYRE_WARMUP_BATCH_SIZES=2,4,6` wouldn't work out too well.)

The model and cache configs are also updated in a similar fashion below